index.js

```diff
@@ -1,3 +1,4 @@
 export function hiMom() {
-  return "Hi, mom!";
+  const language = window !== "undefined" ? window.navigator.language : "en-US";
+  return language === "en-US"  ?  "Hi, mom!" : "Hi, mum!";
```

Comment:
What about `en-AU`?

Reply:
i think, we can add options object to hiMom() with locale option. by default, it will be window.navigator.language || "en_US". final string will be taken from json file with format "{locale}": "{string in that language}"

Reply:
but that means, library will contain a big json file with all the locales

Reply:
> What about `en-AU`?

I'm in `en-AU` and we use `mum`.  It is kind of the USA that are the outliers in the English speaking world and my code handles that.

Reply:
`en-AU` should be `¡ɯnɯ 'ᴉH`.